cluster/cluster_config.go

```diff
@@ -21,89 +21,83 @@ import (
 	"time"
 
 	"github.com/hazelcast/hazelcast-go-client/internal"
+	"github.com/hazelcast/hazelcast-go-client/types"
 )
 
-type Config struct {
-	SecurityConfig       SecurityConfig
-	SSLConfig            SSLConfig
-	Name                 string
-	HazelcastCloudConfig HazelcastCloudConfig
-	Addrs                []string
-	InvocationTimeout    time.Duration
-	HeartbeatInterval    time.Duration
-	HeartbeatTimeout     time.Duration
-	ConnectionTimeout    time.Duration
-	DiscoveryConfig      DiscoveryConfig
-	RedoOperation        bool
-	SmartRouting         bool
-}
+const (
+	defaultName    = "dev"
+	defaultAddress = "127.0.0.1:5701"
+)
 
-func NewConfig() Config {
-	return Config{
-		Name:                 "dev",
-		Addrs:                []string{"127.0.0.1:5701"},
-		SmartRouting:         true,
-		ConnectionTimeout:    5 * time.Second,
-		HeartbeatInterval:    5 * time.Second,
-		HeartbeatTimeout:     60 * time.Second,
-		InvocationTimeout:    120 * time.Second,
-		SecurityConfig:       NewSecurityConfig(),
-		SSLConfig:            NewSSLConfig(),
-		HazelcastCloudConfig: NewHazelcastCloudConfig(),
-		DiscoveryConfig:      NewDiscoveryConfig(),
-	}
+type Config struct {
```

Comment:
Why this is called `Config` while others are called `XxxConfig`

Reply:
It's in the `cluster` package, so `cluster.Config`.